### PR TITLE
Update project metadata

### PR DESCRIPTION
**Pull request by jackvreeken**
- [x] Merge #314 first

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -2,6 +2,57 @@
...
+    { name = "Jack Vreeken", email = "jack@vreeken.me" },
+    { name = "Kent Rutan", email = "gs1150e@icloud.com" },
+]
+requires-python = ">=3.8"
+dependencies = [
+    "numpy >= 1.8.2",
+    "antlr4-python3-runtime == 4.13.*",
+]
+classifiers = [
+    "Development Status :: 4 - Beta",
```
Comment (kentrutan) [inline pyproject.toml:25]: Jumping straight from 1-Planning to 4-Beta in one fell swoop! 😃 I'm thinking more Alpha than Beta from the perspective of Modelica language compliance and definition of the API for general usage in new areas, but you have a better feel for current use cases. We can maybe put some of our thoughts on this into a roadmap in #297.

Comment (jackvreeken) [reply pyproject.toml:25]: Fine, I'll make it Alpha ;)

**Comment by jackvreeken** [comment]
My god, it's still happening! 😭 I can add a delay there of a second or two, but honestly doubt that it will every reliably work...

**Comment by kentrutan** [comment]
Sorry for the bad news ☹️. It's a tricky bugger.

**Pull request merged**